frontend/src/Components/Elements/Table/TableModal.js

```diff
@@ -519,13 +519,7 @@ function TableModal({ chartState, dataSet, columns, isOpen, closeModal }) {
     let [start, end] = dates;
 
     setStartDate(start);
-
-    // Don't allow same month/year selected
-    if (end && start.getDay() === end.getDay()) {
-      end = new Date(end.setMonth(end.getMonth() + 1));
-    } else {
-      setEndDate(end);
-    }
+    setEndDate(end);
```

Comment:
Allows to view data for one month of wanted, say Jan 2022 to view granular data.

Reply:
Only for table data, not supported in graphs